### PR TITLE
1246684 sample template

### PR DESCRIPTION
**Pull request by stephaniehobson**
Small change to an MDN plugin and an upgrade to CKEditor - mostly new language files.

### CODE REVIEW

**Comment by stephaniehobson** [comment]
@jwhitlock You've reviewed ckEditor changes before you're probably best to review this.

File: kuma/static/js/libs/ckeditor/source/plugins/mdn-sampler/plugin.js [hunk 1]
```diff
@@ -32,6 +32,7 @@ CKEDITOR.plugins.add('mdn-sampler', {
         makeElement('pre', gettext('Sample JavaScript Content'), { 'class': 'brush: js' });
 
         // Create the IFrame?
+        makeElement('h3', gettext('Result'));
```
Comment (jwhitlock) [inline kuma/static/js/libs/ckeditor/source/plugins/mdn-sampler/plugin.js]: Is this [bug 1246684](https://bugzilla.mozilla.org/show_bug.cgi?id=1246684)? If so, please split CKEditor upgrade from the change commit next time there's CKEditor work,

**Comment by jwhitlock** [comment]
In my local VM, I followed [the instructions](https://kuma.readthedocs.io/en/latest/ckeditor.html) to reinstall CKEditor and confirm the files were the same.  However, changelog and other files change, which led me to realizing CKEditor 4.5.10 is [still in development](http://dev.ckeditor.com/milestone/CKEditor%204.5.10), and 4.5.9 is the [current released version](http://ckeditor.com/whatsnew).  We shouldn't update until it is released.

In practical tests, [bug 1246684](https://bugzilla.mozilla.org/show_bug.cgi?id=1246684) is fixed - "Insert Code Sample Template" button does insert a new "Result" section. The only issue I had is my inability to write HTML, CSS, and JS correctly the first time.

**Comment by stephaniehobson** [comment]
When I ran the plugin compiler to make the change for bug 1246684 it also ran some portion of the CKEditor upgrade. I was not able to separate those changes from the plugin edits and I was not comfortable submitting those change with the upgrade half done.

I will close this PR and resubmit when CKEditor 4.5.10 is out of development.

**Comment by jwhitlock** [comment]
The ``build.sh`` script looks bad to me. Filed [bug 1275387](https://bugzilla.mozilla.org/show_bug.cgi?id=1275387) to track the work when this becomes urgent.

**Pull request closed without merging**